k6/createPosts.js

```diff
@@ -0,0 +1,63 @@
+import http from 'k6/http';
+
+const config = JSON.parse(open('../config/config.json'));
+const response = JSON.parse(open('../temp_store.json'));
+
+export const options = {
+    vus: config.PostsConfiguration.Count,
```

Comment:
what is the meaning of vus here?

Reply:
virtual users

Reply:
I think we should change the name. Ex. VirtualUsersCount or usersCount

Reply:
discussed